src/testRunner/unittests/tsserver/completions.ts

```diff
@@ -101,7 +101,8 @@ import {
         const localReact: File = {
             path: `${localAtTypes}/react/index.d.ts`,
             content: `import * as PropTypes from 'prop-types';
-`
+export class Component {}
+`,

```

Comment:
The `namespace React` didn't have any runtime values, so it was being treated as type-only. My change actually made this test more accurate (and therefore fail the baseline)! 

_(I also made this change in a few other test files)_

Reply:
Why does this particular test fail? I don't see any .js files in the test.

Reply:
Sorry, what do you mean by "fail"? I don't follow.

The completion request is triggered from `"file": "e:/myproject/src/app.js",` (line 323 right now).

Reply:
(I meant "fail after updating src/services/completions.ts but before updating the test", which I now see wasn't at all clear)

Now that I know to look for src/app.js, I saw it right away. Sorry about the confusion.